### DISCUSSION TITLE
Publishing a collection to the community namespace

### DISCUSSION THREAD

**Discussion by rhysmeister** [ansible-collections/overview#182] [Q&A]
Hello All,

Could anyone help me out with publishing a collection to the community namespace? I've completely forgotten the process and I can't seen to find anything documented. The collection is community.elastic @ https://github.com/ansible-collections/community.elastic/

Cheers,

Rhys

**Comment by tadeboro** [comment]
Community collections are usually published by Zuul. All maintainers have to do it prepare the repo (changelog, galaxy.yml updates, etc.) and then tag the release.

But since I cannot find community.elastic on https://galaxy.ansible.com/community nor on https://dashboard.zuul.ansible.com/t/ansible/projects, I would say that this repo has not been set up yet.

/cc @gundalow

**Comment by Andersson007** [comment]
there's a link "Request namespace" in the section https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#collection-naming
